coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -14,6 +14,10 @@ $(document).ready(function documentReady() {
     items: 1
   });
 
+  $('.course-carousel').on('changed.owl.carousel', function(event) {
+    window.scrollTo(20, 20);
```

Comment:
@Sharonsyra Is there a way to get xpos and ypos from the actual carousel? In a situation where the carousel is not the first item in a lesson card, this will break.

Reply:
Looking into this.

Reply:
@chegejames this is fixed now. Check it out.